<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background:transparent}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/img/bg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="albumImg">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>24</y>
     <width>136</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>136</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>136</width>
     <height>160</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>26</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;font color=grey&gt;&lt;b&gt;T.I.&lt;/b&gt;&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="songName">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>54</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;font color=black&gt;&lt;b&gt;That's All She Wrote&lt;/b&gt;&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>32</y>
     <width>13</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>13</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>13</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/img/pause_normal.png);}
QToolButton:hover{border-image: url(:/img/pause_clicked.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>54</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-04:55</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="seeker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>80</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {

background: #727982;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
width:0px
}

QSlider::handle:horizontal:hover {
width: 0px
}

QSlider::sub-page:horizontal:disabled {
background: #94ad6b;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
width: 0px
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="likeButton">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/like.png);}
QToolButton:hover{border-image: url(:/img/unlike.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="trashButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/trash_can_normal.png);}
QToolButton:hover{border-image: url(:/img/trash_can_clicked.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="nextButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/next_normal.png);}
QToolButton:hover{border-image: url(:/img/next_clicked.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
